code/edgeidentity/src/androidTest/java/com/adobe/marketing/mobile/edge/identity/IdentityECIDHandlingTest.java

```diff
@@ -80,7 +88,14 @@ public void testECID_loadsIdentityDirectECID() throws Exception {
 	@Test
 	public void testECID_whenBothExtensionRegistered_install() throws Exception {
 		// setup
-		registerBothIdentityExtensions(); // no ECID exists before this step
+		registerExtensions(
+			Arrays.asList(
+				MonitorExtension.EXTENSION,
+				Identity.EXTENSION,
+				com.adobe.marketing.mobile.Identity.EXTENSION
```

Comment:
I see this test is failing. We might need to check what is going on here.

Reply:
Ah. Looks like I missed adding a note for this in the description. This is a known regression (the same one that was discussed on slack) that we identified in Identity 2.0 extension (this is being fixed). I did verify that the tests pass with the [patch fix that was made](https://github.com/adobe/aepsdk-core-android/commit/2feaf1f05e25355abaa5afdaee36d512870a08fd) to Identity 2.0 .

Reply:
I see 👍. I missed this discussion.

Reply:
@cacheung The test failure will be addressed later.